solution/backend/regulations/templates/regulations/spa_base.html

```diff
@@ -5,6 +5,12 @@
     <link rel="stylesheet" href="{% static '/vite/index.css' %}?{{ DEPLOY_NUMBER }}" />
 {% endblock %}
 
+{% block outside_vue %}
+    <div id="logoutContainer">
+        {% include "regulations/partials/logout_form.html" %}
+    </div>
+{% endblock %}
+
```

Comment:
the logout form is added to the SPA pages (Search, Statutes, Subjects) and then hidden via CSS.  The form is then submitted programmatically in Vue using a click event.  This allows us to keep the existing Django logout form and use it anywhere on the site without having to rewrite it in Javascript for use in Vue.